examples/tv-casting-app/android/App/app/src/main/jni/com/chip/casting/TargetContentApp.java

```diff
@@ -0,0 +1,13 @@
+/** a.k.a content app like the PV app on a TV */
+public class TargetContentApp {
```

Comment:
```suggestion
public class ContentApp {
```

Consider removing "Target" from the names. Not sure that is a useful prefix to preserve

Reply:
Done!